.github/workflows/trigger-dependent-repo-update.yml

```diff
@@ -0,0 +1,26 @@
+name: Trigger Dependent Repo Update Workflow
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate authentication token
+        id: gen-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.CVAT_BOT_APP_ID }}
+          private-key: ${{ secrets.CVAT_BOT_PRIVATE_KEY }}
+          repositories: ${{ secrets.CVAT_DEPENDENT_REPO }}
+
+      - name: Trigger repository dispatch
+        uses: peter-evans/repository-dispatch@v3
```

Comment:
It would be preferable to pin the version to a full commit hash: https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions